dotCMS/src/main/java/com/dotmarketing/util/PageMode.java

```diff
@@ -166,9 +177,15 @@ private static PageMode getCurrentPageMode(final HttpSession ses) {
     }
 
     private static boolean isNavigateEditMode(final HttpSession ses) {
-        PageMode sessionPageMode = (PageMode) ses.getAttribute(WebKeys.PAGE_MODE_SESSION);
+        PageMode sessionPageMode = ses==null ? DEFAULT_PAGE_MODE : (PageMode) ses.getAttribute(WebKeys.PAGE_MODE_SESSION);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'sessionPageMode' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5106108)